src/routes/view-content/+page.svelte

```diff
@@ -1,432 +1,8 @@
-            await cacheNonSelectedBiblesIfOnline();
-        }
-    }
-
-    async function fetchContentForBibleId(id: number | null) {
-        const index = bibleData?.biblesForTabs.findIndex((bible) => bible.id === id) ?? -1;
-        if (bibleData && bibleData.biblesForTabs[index]) {
-            bibleData.biblesForTabs[index]!.loadingContent = true;
-            const bibleBook = bibleData.biblesForTabs[index];
-            if (bibleBook && $currentBibleSection) {
-                if (!bibleBook.content) {
-                    const content = await fetchBibleContent($currentBibleSection, bibleBook);
-                    // make sure the index hasn't changed
-                    if (bibleData.biblesForTabs[index]!.id === id) {
-                        bibleData.biblesForTabs[index]!.content = content;
-                        populateBibleAudioState();
-                    }
-                }
-            }
-            const currentIndex = bibleData.biblesForTabs.findIndex((bible) => bible.id === id) ?? -1;
-            if (currentIndex >= 0 && bibleData?.biblesForTabs[currentIndex]) {
-                bibleData.biblesForTabs[currentIndex]!.loadingContent = false;
-            }
-        }
-    }
-
-    // in order to make sure content is available when the user goes offline, cache all the Bible data for each
-    // non-selected tab
-    async function cacheNonSelectedBiblesIfOnline() {
-        if ($isOnline && $currentBibleSection && bibleData?.biblesForTabs) {
-            await cacheBiblesForBibleSection(
-                $currentBibleSection,
-                bibleData?.biblesForTabs
-                    .filter(({ id, bookMetadata }) => id !== currentBibleId && bookMetadata !== null)
-                    .map(({ id, bookMetadata }) => ({ ...bookMetadata, bibleId: id })) as BibleBookContentDetails[]
-            );
-        }
-    }
-
-    // Populate the audio state object with key/values like
-    //   bible1 => MultiClipAudioState
-    //   bible2 => MultiClipAudioState
-    function populateBibleAudioState() {
-        if (bibleData) {
-            bibleData.biblesForTabs.forEach((bible) => {
-                if (!Object.keys(multiClipAudioStates).includes(bibleAudioKey(bible.id))) {
-                    const bibleAudioFiles = filterBoolean(
-                        bible.content?.chapters?.map(({ audioData }) => audioData)
-                    ).map((data) => ({
-                        url: data.url,
-                        startTime: data.startTimestamp || 0,
-                        endTime: data.endTimestamp,
-                        type: audioFileTypeForBrowser(),
-                    }));
-                    if (bibleAudioFiles.length) {
-                        multiClipAudioStates[bibleAudioKey(bible.id)] = createMultiClipAudioState(bibleAudioFiles);
-                    }
-                }
-            });
-        }
+    if (browser) {
+        goto('/view-content/-/-');
```

Comment:
This page is here for backwards compatibility. I will redirect to the new page URL.